docs/introduction/media/media.md

```diff
@@ -4,13 +4,16 @@
 - [M3: Uber’s Open Source, Large-scale Metrics Platform for Prometheus](https://eng.uber.com/m3/) By Rob Skillington - Aug 7, 2018.
 
 ## Recorded Talks
-- [Panel Discussion: Prometheus Long-Term Storage Approaches](https://youtu.be/VvJx0WTiGcA?t=23530) including highlights of the M3 stack by Nikunj Aggarwal - Aug 9, 2018.
+- [KubeCon Seattle 2018 Keynote: Smooth Operator♪: Large Scale Automated Storage with Kubernetes](
+https://www.youtube.com/watch?v=aDFm5KaTaOk) By Celina Ward & Matt Schallert - Dec 13, 2018.
 
-- [Putting billions of time series to work at Uber with autonomous monitoring](https://vimeo.com/274821002) By Prateek Rungta - Jun 6, 2018. [Slides](http://bit.ly/m3db-monitorama2018)
+- [Learnings, patterns and Uber’s metrics platform M3, open sourced as a Prometheus long term storage backend](https://www.youtube.com/watch?v=mrq-TBXpztU) By Rob Skillington - Nov 5, 2018. [Slides](https://www.slideshare.net/NETWAYS/osmc-2018-learnings-patterns-and-ubers-metrics-platform-m3-open-sourced-as-a-prometheus-long-term-storage-backend-by-rob-skillington)
 
 - [Adventures in building a high-volume Time-Series Database](https://www.youtube.com/watch?v=W9duNO2dauc) By Richard Artoul & Prateek Rungta - Nov 4, 2018.
 
-## Upcoming Talks
-- [Keynote: Smooth Operator♪: Large Scale Automated Storage with Kubernetes](https://kccna18.sched.com/event/Gsxn/keynote-smooth-operator-large-scale-automated-storage-with-kubernetes-celina-ward-software-engineer-matt-schallert-site-reliability-engineer-uber) By Celina Ward & Matt Schallert - Dec 13, 2018.
+- [Panel Discussion: Prometheus Long-Term Storage Approaches](https://youtu.be/VvJx0WTiGcA?t=23530) including highlights of the M3 stack by Nikunj Aggarwal - Aug 9, 2018.
```

Comment:
We also got a lightning talk at Promcon: https://www.youtube.com/watch?v=_L5RjJ7MVv4&feature=youtu.be&t=1675.